### DISCUSSION TITLE
Where's the marketplace for commercial modules?

### DISCUSSION THREAD

**Discussion by Tony20221** [oqtane/oqtane.framework#2760] [General]
Where's the marketplace for commercial modules? Specifically where one can sell modules using some kind of integrated licenecing mechanism. I recall reading about coming out soon like 2 years ago and I haven't seen it. 

After 4-5 years, Oqtane still doesn't have any good modules that are production ready instead of POC ones.
When DNN came out, it didn't take that long for quality modules to come out and for someone to create a marketplace for modules and themes. Talking about snowcovered.com which DNN the company acquired later.

The more quality modules exist, the faster the adoption of Oqtane will be. 
I am also wondering why DNN module authors are not porting their modules to Oqtane. Most probably because the number of Oqtane users is still pretty low. The number of users is low because there aren't enough Oqtane users. This feel like a catch-22 situation.

(I am aware of 2sxc)

**Comment by sbwalker** [comment]
A commercial marketplace is coming soon... I will post a separate announcement to provide more context shortly.

**Comment by vnetonline** [comment]
@sbwalker can we have an update on what’s happening in this space … are we any closer to selling modules and themes developed for Oqtane ??

**Reply by sbwalker** [reply]
The commercial marketplace is actually my main focus now that 4.0 has been released.
